app/services/notifications/setup/fetch-recipients.service.js

```diff
@@ -84,8 +85,7 @@ FROM (
       ELSE 'Letter - returns to'
     END) AS message_type,
     (NULL) AS recipient,
-    LOWER( concat_ws(',', contacts->>'salutation', contacts->>'forename', contacts->>'initials', contacts->>'name', contacts->>'addressLine1', contacts->>'addressLine2', contacts->>'addressLine3', contacts->>'addressLine4', contacts->>'town', contacts->>'county', contacts->>'postcode', contacts->>'country'))
-      AS contact,
+    contacts as contact,
```

Comment:
Seeing issues with some fields not being lowercased and showing as duplicates then they are (when lowercased properly)